apps/gbfs/test/gbfs/page_cache_test.exs

```diff
@@ -0,0 +1,64 @@
+defmodule GBFS.PageCacheTest do
+  # NOTE: this test will be polluted by other controller tests since it touches the same cache.
+  # We set async to false to avoid that.
+  use GBFS.ConnCase, async: false
+  use GBFS.ExternalCase
+  import Mock
+
+  def run_query(conn, url) do
+    response =
+      conn
+      |> get(url)
+
+    assert response.status == 200
+    assert response.resp_body |> Jason.decode!() |> Map.has_key?("data")
+
+    # NOTE: not using json_response directly because it currently does not catch bogus duplicate "charset"
+    assert response |> get_resp_header("content-type") == ["application/json; charset=utf-8"]
+  end
+
+  test "caches HTTP 200 output", %{conn: conn} do
+    # simulate a clean start, otherwise other tests will have filled the cache
+    Cachex.clear(:gbfs)
```

Comment:
Come to think of it, it is a broader problem even, because the other tests potentially running in parallel due to `async: true` (I haven't checked) will pollute each other.

Maybe it would be better to disable the caching during tests in general (except this specific one), with a form of configuration.

In all cases, this specific test needs to be 100% sure that the cache entry is cleared at the start.

I think I'll need more time to think about that, but it is a very good note.

Reply:
I am working on a solution to have cache disabled by default on all tests, unless explicitly activated, because it will otherwise lead to difficult to track bugs.

Reply:
I'll keep it that way for now, because I need to be sure the cache is empty at start, but I have implemented a dynamic config change to disable caching for tests. Good enough for now I think!